### ISSUE TITLE
Distorted UI

### ISSUE THREAD

**Issue by dahall** [dahall/AeroWizard#24]
When using medium magnification, 125%, "Next" and "Cancel" buttons are not displayed properly - they are partially hidden at their bottom as in attached image. This is serious bug. I'm programmer myself, and I know how important is to test the UI in all three magnifications: 100%, 125% and 150%.
I have Windows 8.1 64 bit.

Another minor inconvenience is when adding your controls to Visual Studio 2013 Toolbox, they are displayed rogue - without suitable image.

Thank you.

Originally posted: 2015-06-06T14:16:48

**Comment by dahall** [comment]
P.S.
I am using .NET 4 build.

Originally posted: 2015-06-06T14:21:44

**Comment by dahall** [comment]
This only comes into effect when an application is declared as DPI-aware in its Manifest (which it really should be to avoid a blurry auto-scaled look).

This StackOverflow article contains a bunch of helpful tips for fixing DPI-scaling issues in WinForms:
[How to write WinForms code that auto-scales to system font and dpi settings](http://stackoverflow.com/questions/22735174/how-to-write-winforms-code-that-auto-scales-to-system-font-and-dpi-settings)

From a quick look at [WizardControl.designer.cs](https://aerowizard.codeplex.com/SourceControl/latest#AeroWizard/WizardControl.designer.cs) simply adding `this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;` might already do the trick.

Originally posted: 2015-06-20T13:20:42

**Comment by dahall** [comment]
This "effect" I've experienced in a new auto-generated Windows Forms project with default manifest, which is not DPI-aware. My own buttons, which I add to "Next", are fine, only "Next and "Cancel" aren't.
AutoScaleMode is the property of the containing form, not the Wizard control, and equals to "Font" by default.
I've tried to play with the source code and failed - those buttons always stay the same height.

Originally posted: 2015-06-20T15:50:55

**Comment by dahall** [comment]
Thanks for the suggestions. I have posted updated code and a testing copy of the library here. I will release 2.0.10 after a little more testing on my side and hearing that I have resolved this issue from one of you.

__demidov71__: Please confirm that I have also corrected the toolbox icon problem.

Originally posted: 2015-06-22T13:33:51

**Comment by dahall** [comment]
Negative. Upon verification of posted here 2.0.10 beta .NET 4 build on Windows 8.1 64-bit English 125% standard magnification, same artifact is shown. It seems the entire footer line is not scaled. I attach the screenshot with Wizard buttons and mine added to them. You see, mine is displayed normally, while built-in are not. There is not enough height for them in the footer. Release 2.0.0 is Ok, though.

Originally posted: 2015-06-24T10:55:34

**Comment by dahall** [comment]
I've uploaded a new beta. Will you try this one?

Originally posted: 2015-06-24T15:12:18

**Comment by dahall** [comment]
Yes, this one, posted here, is good - no scaling artifact. But the "stable" version on site still has it.
No icons in VS toolbox, though.

Originally posted: 2015-06-25T09:29:50

**Comment by dahall** [comment]
I will post this new code under version 2.0.11. Thanks for your testing.

After a great deal of research I have determined that when using NuGet packages which provide different control assemblies for different versions of .NET, you cannot automatically register those controls into the toolbox. I have confirmed that all the controls in this project's assemblies can be manually added to the toolbox and they each have distinct icons when this is done.

If you know a way to get this done automatically, without a separate installer package, I'd like to know.

Originally posted: 2015-06-25T13:16:37
